src/theme/globalStyles.ts

```diff
@@ -22,20 +22,23 @@ const GlobalStyle = createGlobalStyle`
 
   h1 {
     color: ${colors.secondaryFontColor};
+    font-family: ${fonts.montserrat};
```

Comment:
You could make a block for shared styles, like:
```
h1, h2, h3, h4, h5, h6 {
    font-family: ${fonts.montserrat};
}
```
And then not have to repeat it for them here

Reply:
Let me know if this still needs to be updated